Fix TestSequencer()

## Steps to reproduce the problem

TestSequencer/basic_flow:
https://github.com/voedger/voedger/actions/runs/14152882111/job/39648412702
```
--- FAIL: TestSequencer (0.00s)
      --- FAIL: TestSequencer/basic_flow (0.00s)
          impl_test.go:133: 
              	Error Trace:	/home/runner/work/voedger/voedger/pkg/isequencer/impl_test.go:[133](https://github.com/voedger/voedger/actions/runs/14152882111/job/39648412702#step:7:135)
              	Error:      	Not equal: 
              	            	expected: 0x0
              	            	actual  : 0xc8
              	Test:       	TestSequencer/basic_flow
  FAIL
```

--- a/pkg/isequencer/impl_test.go
+++ b/pkg/isequencer/impl_test.go
@@ -48,7 +48,6 @@ func TestSequencer(t *testing.T) {
 		seq.Flush()
 
 		cleanup()
-		seq.(*sequencer).flusherWG.Wait()
 
 		nums, err := storage.ReadNumbers(1, []SeqID{1})
 		require.NoError(err)

